environment_py3_win.yml

```diff
@@ -19,10 +19,11 @@ dependencies:
   - scipy>=0.16.0
   - tqdm
   - xarray>=0.5.1
-  - dask>=2.0
+  - dask<=2022.9.0
   - cftime>=1.3.1
   - ipykernel
   - pytest
   - nbval
   - pykdtree
+  - cartopy
```

Comment:
See above, I suggest not to make cartopy an explicit dependency (as we had not done before)
```suggestion
```